pubspec.lock

```diff
@@ -101,7 +101,7 @@ packages:
       name: pedantic
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.9.0"
+    version: "1.8.0+1"
```

Comment:
@alessandrofavero not sure why this version is being downgraded? 1.9 is latest in pub.dev - are you on latest flutter stable channel?

Reply:
Yes I am. That's weird I didn't touch pubspec.yaml other than adding the font. I'll fix it.

Reply:
All fixed in the latest commit.